extensions/wikia/PageHeader/Language.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jakubjastrzebski
+ * Date: 05/06/2017
+ * Time: 10:36
+ */
+
+namespace Wikia\PageHeader;
+
+use \WikiaApp;
+
+class Language {
+
+	public $currentLangName;
+	public $languages;
+	public $language_urls;
+
+	public function __construct( WikiaApp $app ) {
+		global $wgContLanguageCode, $wgTitle;
+
+		$this->currentLangName = \Language::getLanguageName( $wgContLanguageCode );
+		$this->languageList = $this->handleLanguages( $app );
+
+		//$request_language_urls = $this->request->getVal( 'request_language_urls' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)